tests/integration/ts_runtime_test.go

```
@@ -0,0 +1,67 @@
+	if _, err := os.Stat(tsDir); os.IsNotExist(err) {
+		t.Skip("testdata/golden/ts not found")
+	}
+
+	// Check if npm is available
+	if _, err := exec.LookPath("npm"); err != nil {
+		t.Skip("npm not found, skipping TypeScript runtime test")
+	}
+
+	// Copy to temp dir to avoid mutating testdata (exclude node_modules/dist)
+	tmpDir := t.TempDir()
+	cpCmd := exec.Command("rsync", "-a", "--exclude=node_modules", "--exclude=dist", tsDir+"/", tmpDir+"/")
```

golangci-lint (2.12.2)

[medium] G204: Subprocess launched with a potential tainted input or cmd arguments

(gosec)